contracts/.solhint.json

```diff
@@ -3,11 +3,10 @@
   "plugins": ["prettier"],
   "rules": {
     "code-complexity": ["error", 7],
-    "compiler-version": ["error", "^0.6.12"],
+    "compiler-version": ["warn", "^0.6.12"],
     "const-name-snakecase": "off",
     "constructor-syntax": "error",
     "func-visibility": ["error", { "ignoreConstructors": true }],
```

Comment:
(arbitrary comment location)

The rounding doesn't properly show the toll in the total on the send page. Maybe if we use the same rounding function used on the withdrawal page for fees, we'll get what we need here?

<img width="507" alt="Screen Shot 2022-03-25 at 4 26 40 PM" src="https://user-images.githubusercontent.com/1479802/160201431-cdf41e00-fcf9-4bea-ac56-5cd70e4c530d.png">

Reply:
hey @davidlaprade, rounding is looking great here, but now on the send page, if I type a decimal w/o the leading zero, I get an error. This seems to be a regression.

<img width="500" alt="Screen Shot 2022-04-01 at 1 31 05 PM" src="https://user-images.githubusercontent.com/1479802/161313421-db5790a7-bd2b-46ae-b829-a373d0cd590e.png">

<img width="754" alt="Screen Shot 2022-04-01 at 1 31 25 PM" src="https://user-images.githubusercontent.com/1479802/161313430-d86274f3-6b3e-4acc-8503-c8f6a05e20a3.png">

Reply:
Nice catch, thanks! Fixed with https://github.com/ScopeLift/umbra-protocol/pull/293/commits/227c5e8ba03799e35473b83379477404eab2d501

Reply:
Working now, thanks @davidlaprade